db/migrate/20211230033135_add_fields_to_organization.rb

```diff
@@ -0,0 +1,6 @@
+class AddFieldsToOrganization < ActiveRecord::Migration[6.1]
+  def change
+    add_column :organizations, :repackage_essentials, :boolean
+    add_column :organizations, :distribute_monthly, :boolean
```

Comment:
Generally a good idea to make booleans `null: false, default: false` because otherwise you have to deal with null booleans, which don't make a lot of sense :)